<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SQLData</class>
 <widget class="QDialog" name="SQLData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Data Display</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../usr/share/machinetrans/resources/logo.png</normaloff>../../../../usr/share/machinetrans/resources/logo.png</iconset>
  </property>
  <widget class="QLabel" name="titleLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>780</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Machine Translation Data Display</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="quitBttn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>520</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findBttn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>520</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listDisplay">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>780</width>
     <height>440</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="delBttn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>520</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editBttn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="helpBttn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sortBttn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>520</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Sort</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>helpBttn</tabstop>
  <tabstop>listDisplay</tabstop>
  <tabstop>editBttn</tabstop>
  <tabstop>delBttn</tabstop>
  <tabstop>findBttn</tabstop>
  <tabstop>quitBttn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
